src/Middleware/Authorization.ts

```diff
@@ -13,10 +16,23 @@ export async function AuthorizeUser(
     try {
         const token = (req.headers.authorization || req.headers.Authorization) as string;
         const authorizedUser = await jwt.verify(token, config.jwtKey) as JwtPayload;
-        if (authorizedUser) {
+        const node = await Node.findByPk(req.params.nodeId)
```

Comment:
This is where your test failed. `req.params` were not sent and therefore from here it would end in the catch block.

It's a good test to realize that middleware wasn't properly tested :))

However, seems like this is complex logic with if else/if. Maybe you should extend passingNodeData middleware and use it always?